index.js

```diff
@@ -119,12 +133,17 @@ function MiFlowerCarePlugin(log, config) {
 
             if (that.humidityAlert) {
                 that.humidityAlertService.getCharacteristic(Characteristic.StatusLowBattery)
-                    .updateValue(data.batteryLevel <= that.lowBatteryWarningLevel ? Characteristic.StatusLowBattery.BATTERY_LEVEL_LOW : Characteristic.StatusLowBattery.BATTERY_LEVEL_NORMAL);
+                    .updateValue(data.moisture <= that.humidityAlertLevel ? Characteristic.StatusLowBattery.BATTERY_LEVEL_LOW : Characteristic.StatusLowBattery.BATTERY_LEVEL_NORMAL);
```

Comment:
These lines are for the low battery status of the humidityAlertService. Here, moisture should not be used. The previous behavior was correct. The same applies to the other changes made.